pom.xml

```diff
@@ -145,7 +145,7 @@
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>exec-maven-plugin</artifactId>
-          <version>1.6.0</version>
+          <version>3.0.0</version>
```

Comment:
The CI (and tests) does not validate this change. I confirmed the command below works with 3.0.0:
```
~/cloud-opensource-java/dashboard $ mvn exec:java -Dexec.arguments="-f ../boms/cloud-oss-bom/pom.xml"
```